packages/scss/src/components/index.scss

```diff
@@ -51,6 +51,9 @@
 @forward 'navside'; // 15 Ko
 @forward 'form'; // 25 Ko
 @forward 'tableFixed'; // 33 Ko
+@forward 'tabs';
+
+// caution: the two components cannot be loaded at the same time
```

Comment:
Are the two components `grid` and `tabs`?

It might be a problem if another `@forward` gets inserted in between. I propose:

`// caution: grid and tabs components cannot be loaded at the same time.`

Reply:
It's a former comment referring to `grid` and `gridLegacy` imports which is kind of unclear due to lint formatting 🙈